src/locales/en-US/strings.ftl

```diff
@@ -108,6 +108,9 @@ onboarding_modal_username = Getting started
   .username_space = Username must not contain spaces
   .username_chars = Username must only contain latin characters a-z or digits 0-9
 
+onboarding_modal_profile_error =
+  .profile_error = Error in creating profile please reach out in <a data-l10n-name="iam_slack">#iam</a> Slack or at <a data-l10n-name="support_email">people-mozilla-org-support@mozilla.com</a>
```

Comment:
I feel these are two sentences, so their should be some interpunction before "please" – probably a peiod.

Reply:
:+1: Good idea!